tests/test_types.py

```diff
@@ -65,9 +65,83 @@ def test_language(self) -> None:
         view = self._make_view("Packages/Python/Python.sublime-syntax", "hello.py")
         self.assertFalse(selector.matches(view))
 
-    def test_pattern(self) -> None:
+    def test_pattern_basics(self) -> None:
         selector = DocumentSelector([{"language": "html", "pattern": "**/*.component.html"}])
         view = self._make_view("Packages/HTML/HTML.sublime-syntax", "index.html")
         self.assertFalse(selector.matches(view))
         view = self._make_view("Packages/HTML/HTML.sublime-syntax", "components/foo.component.html")
         self.assertTrue(selector.matches(view))
+
+    def _make_html_view(self, file_name: str) -> sublime.View:
+        return self._make_view("Packages/HTML/HTML.sublime-syntax", file_name)
+
+    def test_pattern_asterisk(self) -> None:
+        """`*` to match one or more characters in a path segment"""
+        selector = DocumentSelector([{"language": "html", "pattern": "a*c.html"}])
+        # self.assertFalse(selector.matches(self._make_html_view("ac.html")))
```

Comment:
I'm unsure about the wording in the spec here. The wording says

> one or more characters

but in my experience, `*` also matches zero characters.